### PR TITLE
Aggregator tests update

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Ensure the aggregator tests pass in docker env also for automated run in corresponding repository's CI

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Refactor (refactoring code, removing useless files)

## Testing steps

Run tests

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary

### CODE REVIEW

File: features/steps/insights_results_aggregator.py [hunk 1]
```diff
@@ -29,9 +29,16 @@ def run_insights_results_aggregator_with_flag(context, flag):
     start_aggregator(context, flag, environment)
 
 
+@when("I remove all Insights Results Aggregator environment variables")
+def remove_insights_results_aggregator_env_vars(context):
+    context.no_IRA_environment = {key: value for (key, value) in os.environ.copy().items() if
```
Comment (tisnik) [inline features/steps/insights_results_aggregator.py:34]: it does not do what was written in @when :)

Comment (epapbak) [reply features/steps/insights_results_aggregator.py:34]: True, not the best name for what it actually does.

"I store current environment without Insights Results Aggregator environment variables", maybe?

File: features/steps/insights_results_aggregator.py [hunk 2]
```diff
@@ -120,7 +127,7 @@ def check_actual_configuration_for_aggregator(context):
     assert "CertPath" in context.output[6], "Caught output: {}".format(context.output)
 
 
-@when("I migrate aggregator database to version #{version:n}")
+@when("I migrate aggregator database to version #{version}")
```
[the remark this answers is not in the document]
Comment (epapbak) [reply features/steps/insights_results_aggregator.py:130]: can't use `#latest` this way. It's not the best solution, but I didn't want to refactor the step and have one that expects `#version` and another with `version`, so I went with `#latest` here.

[the remark this answers is not in the document]
Comment (tisnik) [reply features/steps/insights_results_aggregator.py:130]:
```
@when("I migrate aggregator database to latest version")
def foobar(context, version='latest'):
    pass
```

[the remark this answers is not in the document]
Comment (epapbak) [reply features/steps/insights_results_aggregator.py:130]: works for me =)

[the remark this answers is not in the document]
Comment (epapbak) [reply features/steps/insights_results_aggregator.py]: updated =)

File: features/steps/insights_results_aggregator.py (hunk 1 above)
Comment (epapbak) [reply features/steps/insights_results_aggregator.py:34]: updated

**Review by juandspy — APPROVED** [review]

**Review by Bee-lee — APPROVED** [review]

**Pull request merged**